### PR TITLE
WINUI-2979 Prepare and include the UG document for DateFilter support for Filter Row

### PR DESCRIPTION
**Pull request by sethuramkumar**
### Description ###
Prepare and include the UG document for DateFilter support for Filter Row

### CODE REVIEW

File: winui/DataGrid/FilterRow.md [hunk 1]
```diff
@@ -155,6 +172,9 @@ When string value is bounded to the {{'[GridColumn](https://help.syncfusion.com/
 <td>
 When Boolean type is bounded to the {{'[GridColumn](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.GridColumn.html)'| markdownify }}, then <code>CheckBox</code> editor is loaded in {{'[FilterRowCell](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.RowFilter.GridFilterRowCellRenderer-2.html#Syncfusion_UI_Xaml_DataGrid_RowFilter_GridFilterRowCellRenderer_2_FilterRowCell)'| markdownify }}.
```
Comment (Chrispineagunja) [inline winui/DataGrid/FilterRow.md:173]: When the Boolean type is bounded to the {{'[GridColumn](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.GridColumn.html)'| markdownify }}, the <code>CheckBox</code> editor is loaded in the {{'[FilterRowCell](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.RowFilter.GridFilterRowCellRenderer-2.html#Syncfusion_UI_Xaml_DataGrid_RowFilter_GridFilterRowCellRenderer_2_FilterRowCell)'| markdownify }}.

File: winui/DataGrid/FilterRow.md [hunk 2]
```diff
@@ -155,6 +172,9 @@ When string value is bounded to the {{'[GridColumn](https://help.syncfusion.com/
...
 </td>
+<td>
+When DateTimeOffset type is bounded to the {{'[GridDateColumn](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.GridDateColumn.html)'| markdownify }}, then <code>Date</code> editor is loaded in {{'[FilterRowCell](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.RowFilter.GridFilterRowCellRenderer-2.html#Syncfusion_UI_Xaml_DataGrid_RowFilter_GridFilterRowCellRenderer_2_FilterRowCell)'| markdownify }}.
```
Comment (Chrispineagunja) [inline winui/DataGrid/FilterRow.md:176]: When the DateTimeOffset type is bounded to the {{'[GridDateColumn](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.GridDateColumn.html)'| markdownify }}, the <code>Date</code> editor is loaded in the {{'[FilterRowCell](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.RowFilter.GridFilterRowCellRenderer-2.html#Syncfusion_UI_Xaml_DataGrid_RowFilter_GridFilterRowCellRenderer_2_FilterRowCell)'| markdownify }}.

File: winui/DataGrid/FilterRow.md [hunk 3]
```diff
@@ -190,6 +210,19 @@ The default filter condition is Begins With, the below filter conditions are ava
 <td>
 Always equals filter condition will be applied for filtering the CheckBox value.
 </td>
+<td>
+The default filter condition is Equals, the below filter conditions are available in date filter.
```
Comment (Chrispineagunja) [inline winui/DataGrid/FilterRow.md:214]: The default filter condition is Equals, and the following filter conditions are available in the date filter.

File: winui/DataGrid/FilterRow.md (hunk 1 above)
Comment (sethuramkumar) [reply winui/DataGrid/FilterRow.md]: Hi @Chrispineagunja ,

I have modified the content as per the concern provided.

File: winui/DataGrid/FilterRow.md (hunk 2 above)
Comment (sethuramkumar) [reply winui/DataGrid/FilterRow.md]: Hi @Chrispineagunja ,

I have modified the content as per the concern provided.

File: winui/DataGrid/FilterRow.md (hunk 3 above)
Comment (sethuramkumar) [reply winui/DataGrid/FilterRow.md]: Hi @Chrispineagunja ,

I have modified the content as per the concern provided.

**Review by susmitha-sundar — APPROVED** [review]

**Pull request merged**